packages/workflow/test/FilterParameter.test.ts

```diff
@@ -570,6 +570,48 @@ describe('FilterParameter', () => {
 					expect(result).toBe(expected);
 				});
 
+				it.each([
+					{ left: 0, expected: false },
+					{ left: 15, expected: false },
+					{ left: -15.4, expected: false },
+					{ left: NaN, expected: true },
+					{ left: null, expected: true },
```

Comment:
Are these tests needed if the option is hidded for numbers? Also should we add tests for other data types?

Reply:
It's hidden in the expressions because of the bug, this test is about the filter component. But will add tests for the types that are missing (DateTime,boolean)